### PR TITLE
RelocaTE2.v2.0.0

### PR DESCRIPTION
**Pull request by JinfengChen**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/relocate2/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export PATH="$PREFIX/bin:$PATH"
```
Comment (bgruening) [inline recipes/relocate2/build.sh]: This should not be needed.

File: recipes/relocate2/build.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+export PYTHONPATH="$PREFIX/lib/python2.7/site-packages:$PYTHONPATH"
```
Comment (bgruening) [inline recipes/relocate2/build.sh]: This also not ... is this not working by default?

File: recipes/relocate2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package:
+  name: relocate2
+  version: 2.0.0
+
+source:
+  git_url: https://github.com/stajichlab/RelocaTE2.git
```
Comment (bgruening) [inline recipes/relocate2/meta.yaml]: Please use a tarball and a checksum.
This will always pull the latest github and if we rebuild the recipe it will not be the same.

File: recipes/relocate2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    - samtools >=1.3.1
+    - bowtie2 >=2.2.8
+    - bedtools >=2.25.0
+    - bwa >=0.6.2
+    - seqtk >=1.2
+    - pysam >=0.9.1
+    - blat
+  run:
+    - samtools 
+    - bowtie2
```
Comment (bgruening) [inline recipes/relocate2/meta.yaml]: You probably want to have the `>=` here as well.

File: recipes/relocate2/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    - bedtools
+    - bwa
+    - seqtk
+    - pysam
+    - blat
+
+#test:
+#  commands:
```
Comment (bgruening) [inline recipes/relocate2/meta.yaml]: A test is strictly needed in bioconda.

File: recipes/relocate2/build.sh [hunk 6]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir $PREFIX/scripts
+cp ./scripts/*.p* $PREFIX/scripts
+cp ./scripts/*.p* $PREFIX/bin
+cp ./README.md $PREFIX
+cp ./test_data.tar.gz $PREFIX
```
Comment (bgruening) [inline recipes/relocate2/build.sh]: How big is this and is this really needed for a package?

File: recipes/relocate2/meta.yaml (hunk 5 above)
Comment (JinfengChen) [inline recipes/relocate2/meta.yaml]: I have problem when building the package offline. The testing failed and says "run_test.sh: line 3: relocaTE2.py: command not found".  I am not sure I understand how the testing works. But it seems building and installation are fine. Can you give me some suggestion how I should modify the testing?
Thanks.

Jinfeng

File: recipes/relocate2/build.sh [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+echo "bwa=$PREFIX/bin/bwa" >> $PREFIX/CONFIG
+echo "bowtie2=$PREFIX/bin/bowtie2" >> $PREFIX/CONFIG
+echo "bowtie2_build=$PREFIX/bin/bowtie2-build" >> $PREFIX/CONFIG
+echo "blat=$PREFIX/bin/blat" >> $PREFIX/CONFIG
```
Comment (bgruening) [inline recipes/relocate2/build.sh]: I guess the problem is here $PREFIX will be a full path and this will not be the same on all systems.
Is this file really needed?

File: recipes/relocate2/build.sh [hunk 8]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir $PREFIX/scripts
+cp ./scripts/*.p* $PREFIX/scripts
+cp ./scripts/*.p* $PREFIX/bin
+cp ./README.md $PREFIX
```
Comment (bgruening) [inline recipes/relocate2/build.sh]: Not needed I guess.

File: recipes/relocate2/build.sh [hunk 9]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir $PREFIX/scripts
+cp ./scripts/*.p* $PREFIX/scripts
```
Comment (bgruening) [inline recipes/relocate2/build.sh]: Is $PREFIX/scripts needed? Can you just copy Python files to bin.

**Comment by JinfengChen** [comment]
I tried using bin. Still get errors. The problem is in '_t_env'. None of
the compiled tools or Python scripts installed or copied into it.  '_b_env'
is fine.
Thanks.

Jinfeng

2016年11月10日星期四，Björn Grüning <notifications@github.com> 写道：
